actioncable/lib/action_cable/connection/base.rb

```diff
@@ -95,8 +95,8 @@ def transmit(cable_message) # :nodoc:
       end
 
       # Close the WebSocket connection.
-      def close
-        websocket.close
+      def close(code = nil, reason = nil)
+        websocket.close(code, reason)
       end
```

Comment:
Good case for keyword arguments here: `def close(code: nil, reason: nil)`

Reply:
Although it is already implemented, I don't see any existing test for parameters.
https://github.com/rails/rails/blob/master/actioncable/lib/action_cable/connection/client_socket.rb#L90

https://github.com/rails/rails/blob/master/actioncable/test/connection/client_socket_test.rb
Also websocket class seems to have no tests at all.

I can't figure out how it could be tested though.